src/compressAlignment.hpp

```diff
@@ -77,26 +77,32 @@ that can be represented                    3       7      511   -2^19   -2^31
 
 // Standard library.
 #include "string.hpp"
-#include "span.hpp"
 
 
 namespace shasta {
     void compress(const Alignment&, string&);
-    void decompress(span<const char>, Alignment);
+    void decompress(const string&, Alignment&);
```

Comment:
We also need the signature with `span` because the bytes will come from a `MemoryMapped::VectorOfVectors`. That can be the lower level routine. The overload that takes a `const string& s` can call that passing `span<const char>(s.begin(), s.begin() + s.size())`.

Reply:
Yup. Will add it.